app/javascript/components/key-performance-indicators/components/goal-progress-per-need/component.jsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { useI18n } from "components/i18n";
+import makeStyles from "@material-ui/styles/makeStyles";
+import TablePercentageBar from "../table-percentage-bar";
+import KpiTable from "../kpi-table";
+import asKeyPerformanceIndicator from "../as-key-performance-indicator";
+
+const GoalProgressPerNeed = ({ data, identifier }) => {
+  const i18n = useI18n();
+  const css = makeStyles({
+    root: {
```

Comment:
Create a stylesheet for this. Don't include css in components if you can help it

Reply:
Could you point me at some docs or explain to me how? I'm not sure how we use css in js to override theme stuff from a style sheet.